devicetypes/smartthings/smartweather-station-tile.src/smartweather-station-tile.groovy

```diff
@@ -418,9 +431,41 @@ private localDate(timeZone) {
     df.format(new Date())
 }
 
-private send(map) {
-    log.debug "WUSTATION: event: $map"
+// Create the new custom capability event if needed,
+// but also send a legacy custom event for any DM-backed SmartApps using them.
+private send(Map map) {
+    def eventConversion = [
+            "localSunrise": "stsmartweather.astronomicalData.localSunrise",
+            "localSunset": "stsmartweather.astronomicalData.localSunset",
+            "city": "stsmartweather.astronomicalData.city",
+            "timeZoneOffset": "stsmartweather.astronomicalData.timeZoneOffset",
+            "weather": "stsmartweather.weatherSummary.weather",
+            "wind": "stsmartweather.windSpeed.wind",
+            "windVector": "stsmartweather.windDirection.windVector",
+            "weatherIcon": "stsmartweather.weatherSummary.weatherIcon",
+            "forecastIcon": "stsmartweather.weatherForecast.forecastIcon",
+            "feelsLike": "stsmartweather.apparentTemperature.feelsLike",
+            "percentPrecip": "stsmartweather.precipitation.percentPrecip",
+            "alert": "stsmartweather.weatherAlert.alert",
+            "alertKeys": "stsmartweather.weatherAlert.alertKeys",
+            "sunriseDate": "stsmartweather.astronomicalData.sunriseDate",
+            "sunsetDate": "stsmartweather.astronomicalData.sunsetDate",
+            "lastUpdate": "stsmartweather.smartWeather.lastUpdate",
+            "uvDescription": "stsmartweather.ultravioletDescription.uvDescription",
+            "forecastToday": "stsmartweather.weatherForecast.forecastToday",
+            "forecastTonight": "stsmartweather.weatherForecast.forecastTonight",
+            "forecastTomorrow": "stsmartweather.weatherForecast.forecastTomorrow"
+        ]
+
+    //log.trace "WUSTATION: event: $map"
```

Comment:
Could remove the commented out code

Reply:
🤷‍♂ I suppose. I figured that I would leave it in in case someone wants to verify any issues that might come up in the future, but commented out so as not to have production code logging for every event. This log statement was the one original to the function.